style.css

```diff
@@ -436,6 +436,28 @@ button:hover {
   display: flex;
   flex-direction: column;
   justify-content: center;
+  position: relative;
+}
+
+.error { 
+  padding: 0;
+  font-size: 80%;
+  color: rgb(219, 0, 0); 
+  border-radius: 0 0 5px 5px;
+  box-sizing: border-box;
+  position: absolute;
+  top: 88%;
+  left: 50%; 
+  border-radius: 2px;
+  font-size: .9rem;
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] I would recommend the error message be more explicit. Something like `Please make email lowercase`.